tasks/config/run.js

```diff
@@ -77,6 +79,22 @@ module.exports = function (grunt) {
       ]
     },
 
+    testUIReleaseServer: {
+      options: {
+        wait: false,
+        ready: /Server running/,
+        quiet: false,
+        failOnError: false
+      },
+      cmd: releaseBinScript,
+      args: [
+        ...stdDevArgs,
+        '--server.port=' + uiConfig.servers.kibana.port,
+        '--elasticsearch.url=' + format(uiConfig.servers.elasticsearch),
+        ...kbnServerFlags,
+      ]
+    },
+
```

Comment:
This is copypasta from `testUIServer`. Is there a better way? Maybe reference the previous object and merge/extend it?